pkgs/development/python-modules/django-filter/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, buildPythonPackage, fetchPypi, python
+, django, django-crispy-forms, djangorestframework, mock, pytz }:
+
+buildPythonPackage rec {
+  pname = "django-filter";
+  version = "2.0.0";
+
```

Comment:
We need to use the `disabled` parameter to respect https://github.com/carltongibson/django-filter/blob/8e917de6c90d3d5512d71bae05dc463ecaebc88b/setup.py#L59

Reply:
Fixed